src/cmd/factory/impl/exchange.go

```diff
@@ -115,7 +115,7 @@ func handleExchangeCalendarEventFactory(cmd *cobra.Command, args []string) error
 		func(id, now, subject, body string) []byte {
 			return mockconnector.GetMockEventWith(
 				User, subject, body, body,
-				now, now, false)
+				now, now, `null`, `[]`, false)
```

Comment:
not important here, but fyi, the grave-tics in go are primarily useful for constructing escaped and multiline strings.  Generally you should use `"` unless otherwise needed.  Example here: https://go.dev/play/p/U8VjEN2bSo7